src/components/Navbar/Navbar.js

```diff
@@ -40,6 +40,12 @@ export function Navbar() {
   const { theme } = useContext(ThemeContext);
   const { t } = useTranslation();
   const [currentLanguage, setCurrentLanguage] = useState('pl');
+  const scrollToProjects = () => {
+    window.scrollTo({
+      top: 1300,
```

Comment:
Punkt, gdzie się scrolluje powinien być określany dynamicznie, bo jak wysoko od góry strony zależy od ilości tekstu w poprzednich sekcjach oraz od rozdzielczości ekranu.

Reply:
tutaj prawdopodobnie by starczyło scrollIntoView i referencja po idku lub klasie do tego elementu z behavior: 'smooth' bez wyliczania, ale trzeba to sprawdzić @seba1288